docs/5.1. Data Model - Identifier Mapping.md

```diff
@@ -22,26 +22,26 @@ All resources shown above are part of the JT-NM reference architecture. Each log
 
 A new Source is created in the following circumstances:
 
--   At the input of essence into a networked media enviroment from the 'edge' (ie. a legacy or incompatible protocol)
--   At the point of capture (ie. in a network connected camera or microphone)
--   At the mix point of two or more Flows of essence (ie. an audio mix bus, vision mix bus)
--   Where a multiplexed Source is split into multiple mono-essence Sources
--   Where a set of Flows are combined as a single multiplexed Source
+- At the input of essence into a networked media enviroment from the 'edge' (ie. a legacy or incompatible protocol)
+- At the point of capture (ie. in a network connected camera or microphone)
+- At the mix point of two or more Flows of essence (ie. an audio mix bus, vision mix bus)
+- Where a multiplexed Source is split into multiple mono-essence Sources
+- Where a set of Flows are combined as a single multiplexed Source
 
-Whenever a new Source is generated, its immediate parent Source(s) MUST be recorded. This is achieved via a 'parents' attribute in the Node API Sources resource. This contains an array of Source IDs corresponding to the Sources whose Flows have been brought together at this point. For example, a vision mixer which is fading between a Flow from Source A, and a Flow from Source B would indicate \["A", "B"\] in its 'parents' attribute for the duration of this transition.
+Whenever a new Source is generated, its immediate parent Source(s) MUST be recorded. This is achieved via a `parents` attribute in the Node API Sources resource. This contains an array of Source IDs corresponding to the Sources whose Flows have been brought together at this point. For example, a vision mixer which is fading between a Flow from Source A, and a Flow from Source B would indicate ["A", "B"\] in its `parents` attribute for the duration of this transition.
```

Comment:
I see a removed backslash before the open square bracket, but not before the close square bracket, is that right? But not backticking this? Hmm, equiv array below is, so suggestion:

```suggestion
Whenever a new Source is generated, its immediate parent Source(s) MUST be recorded. This is achieved via a `parents` attribute in the Node API Sources resource. This contains an array of Source IDs corresponding to the Sources whose Flows have been brought together at this point. For example, a vision mixer which is fading between a Flow from Source A, and a Flow from Source B would indicate `["<id of A>", "<id of B>"]` in its `parents` attribute for the duration of this transition.
```

Reply:
Strange, I remember changing that one already.